### DISCUSSION TITLE
Immune system model

### DISCUSSION THREAD

**Discussion by agoliaei** [FLAMEGPU/FLAMEGPU2#1024] [General]
Hi,
I was wondering, is the code for immune system model open source?
Thanks for sharing FLAMEGPU2 with the community.

**Comment by mondus** [comment]
Can you be more specific as there as a couple of immune system models which I am aware of using FLAME GPU? I am not sure that any of them are open source but if you can identify which I can put direct you to the author of the code/paper.

**Reply by agoliaei** [reply]
Hi,
Thanks so much for replying. I was wondering about the model the speaker in this video is talking about at around 25:30. 
https://www.youtube.com/watch?v=TTfq_qgfv3w
Following this video I did search for papers, but none comes with the code. As you said now I know they are not open source.
I don't have a specific use case right now for the immune system model. I do have experience in using agent based models in cellular biology and was just curious and wanted to understand better how biological systems are modeled using this platform on GPUs. Not a problem if not open sourced. 
Thanks for replying again.

**Reply by mondus** [reply]
We tend to publish all our models using MIT however the immune system model is not one that is open source. It was developed with a collaborator that retained the IP for the underlying model which made open souring it difficult.

**Reply by agoliaei** [reply]
Thank you for clarifying.
